lib/lost-align.js

```diff
@@ -51,96 +56,91 @@ module.exports = function lostAlignDecl(css, settings) {
           );
         } else if (lostAlign == 'middle-left' || lostAlign == 'left') {
           newBlock(
-            decl,
-            ' > *',
-            ['position', 'top', 'right', 'bottom', 'left', 'transform'],
-            ['absolute', '50%', 'auto', 'auto', '0', 'translate(0, -50%)']
+              decl,
+              ' > *',
+              ['position', 'top', 'right', 'bottom', 'left', 'transform'],
+              ['absolute', '50%', 'auto', 'auto', '0', 'translate(0, -50%)']
           );
         } else if (lostAlign == 'middle-center' || lostAlign == 'center') {
           newBlock(
-            decl,
-            ' > *',
-            ['position', 'top', 'right', 'bottom', 'left', 'transform'],
-            ['absolute', '50%', 'auto', 'auto', '50%', 'translate(-50%, -50%)']
+              decl,
+              ' > *',
+              ['position', 'top', 'right', 'bottom', 'left', 'transform'],
+              ['absolute', '50%', 'auto', 'auto', '50%', 'translate(-50%, -50%)']
           );
         } else if (lostAlign == 'middle-right' || lostAlign == 'right') {
           newBlock(
-            decl,
-            ' > *',
-            ['position', 'top', 'right', 'bottom', 'left', 'transform'],
-            ['absolute', '50%', '0', 'auto', 'auto', 'translate(0, -50%)']
+              decl,
+              ' > *',
+              ['position', 'top', 'right', 'bottom', 'left', 'transform'],
+              ['absolute', '50%', '0', 'auto', 'auto', 'translate(0, -50%)']
           );
         } else if (lostAlign == 'bottom-left') {
           newBlock(
-            decl,
-            ' > *',
-            ['position', 'top', 'right', 'bottom', 'left', 'transform'],
-            ['absolute', 'auto', 'auto', '0', '0', 'translate(0, 0)']
+              decl,
+              ' > *',
+              ['position', 'top', 'right', 'bottom', 'left', 'transform'],
+              ['absolute', 'auto', 'auto', '0', '0', 'translate(0, 0)']
           );
         } else if (lostAlign == 'bottom-center' || lostAlign == 'bottom') {
           newBlock(
-            decl,
-            ' > *',
-            ['position', 'top', 'right', 'bottom', 'left', 'transform'],
-            ['absolute', 'auto', 'auto', '0', '50%', 'translate(-50%, 0)']
+              decl,
+              ' > *',
+              ['position', 'top', 'right', 'bottom', 'left', 'transform'],
+              ['absolute', 'auto', 'auto', '0', '50%', 'translate(-50%, 0)']
           );
         } else if (lostAlign == 'bottom-right') {
           newBlock(
-            decl,
-            ' > *',
-            ['position', 'top', 'right', 'bottom', 'left', 'transform'],
-            ['absolute', 'auto', '0', '0', 'auto', 'translate(0, 0)']
+              decl,
+              ' > *',
+              ['position', 'top', 'right', 'bottom', 'left', 'transform'],
+              ['absolute', 'auto', '0', '0', 'auto', 'translate(0, 0)']
```

Comment:
Yeah seems that way. Might be worth implementing Semistandard as a first step of 7.0.0?